packages/sources/react/src/components/VtmnLink/VtmnLink.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import clsx from 'clsx';
+
+import '@vtmn/css-link';
+import { VtmnLinkSize } from './types';
+
+export interface VtmnLinkProps extends React.ComponentPropsWithoutRef<'a'> {
+  /**
+   * The size of the link.
+   * @default 'medium'
+   */
+  size?: VtmnLinkSize;
+
+  /**
+   * Whether link is standalone or not.
+   * @type {boolean}
+   * @default false
+   */
+  isStandalone?: boolean;
+}
+
+export const VtmnLink = ({
+  isStandalone = false,
+  size = 'medium',
```

Comment:
I would be tented to make an enumeration here. To avoid setting size to 'small' but using VtmnLinkSize.SMALL instead.

Reply:
Same comment i think for consumption that's more difficult, they have to know the enumeration 🤔 
@lauthieb ? :D

Reply:
Yes I understand the argument, even for library side only, it could be nice to don't have "magic words" like "medium" and avoid bugs by securing with enum. I will investigate.